test/ava-files.js

```diff
@@ -104,3 +104,29 @@ test('findFiles - does not return duplicates of the same file', function (t) {
 		t.end();
 	});
 });
+
+test('findFiles - finds the correct files by default', function (t) {
+	var avaFiles = new AvaFiles(['**/ava-files/default-patterns/**']);
```

Comment:
I can do that. Unfortunately I've run out of time. Someone can feel free to branch off of my branch and make the improvements in another PR if it needs to be fixed sooner than I can get to it.